offer-maker/Direct.md

```diff
@@ -6,11 +6,11 @@ Direct should be seen as an implementation that only works for one user, the adm
 
 Direct does many of the same things as MangroveOffer with a few key differences.
 
-**How inbound tokens are deposited to the maker:** After the funds have been transferred from the taker to the Direct contract, it chooses to leave received funds on the contract's balance, until `makerPosthook` is called. The reason for leaving the funds on the contract is that multiple offers posted by this contract may be taken in the same market order. This will result in a cumulative amount of inbound tokens being on the contract's balance. Instead of transferring them during each call to `makerExecute`, it is cheaper in gas to wait until after all offers are taken, and then transfer all the funds left on the contract to the reserve.
+**How inbound tokens are handled (from the taker):** After the funds have been transferred from the taker to the Direct contract, it chooses to leave received funds on the contract's balance, until `makerPosthook` is called. The reason for leaving the funds on the contract is that multiple offers posted by this contract may be taken in the same market order. This will result in a cumulative amount of inbound tokens being on the contract's balance. Instead of transferring them during each call to `makerExecute`, it is cheaper in gas to wait until after all offers are taken, and then transfer all the funds left on the contract to the reserve.
 
-**How outbound tokens are deposited to the taker:** When transferring the funds from the contract to the taker, Direct first tries to check if it itself has the funds, otherwise tries to get the funds from the reserve either by using a router or by using the contract itself. This means that if the contract already has the funds, no extra transfers are needed.
+**How outbound tokens are handled (for the taker):** When transferring the funds from the contract to the taker, Direct first tries to check if it itself has the funds, otherwise tries to get the funds from the reserve either by using a router or by using the contract itself. This means that if the contract already has the funds, no extra transfers are needed.
 
-**Approvals:** When Direct checks if the maker is allowed to set a given address as the reserve, it just checks whether the caller is the admin of the contract. Since the contract can only be used by the admin, there is no need to check anything else.
+**Reserve:** When Direct checks if the maker is allowed to set a given address as the reserve, it just checks whether the caller is the admin of the contract. Since the contract can only be used by the admin, there is no need to check anything else.
```

Comment:
Maker can change its fund reserve. Default is the maker's address, but setting the reserve to be the `Direct` contract itself will result in a more gas efficient trade (no transfer is needed). Any reserve other than maker's address or `Direct` contract needs to approve the `Direct` contract for outbound token transfer.

Reply:
I've added to the explanation for reserve in MangroveOffer, since some of it was not specific for Direct. And i've updated the Directs description of how it uses reserve.